slapp/explorer/settings.py

```diff
@@ -53,3 +53,45 @@
         "color": "#4682B4",
     },
 }
+
+TECHNOLOGIES_RANGES = {
+    "boiler_large": {
+        "potential": "60 [MW]",
```

Comment:
I'd rather have a real integer here and read unit from settings.py and merge them within chart function.
If you do so, you must adapt `get_potential` function as well to return `None` instead of "-" and handle this later on.